app/gui/src/components/ProtectedLayout.vue

```diff
@@ -129,6 +131,8 @@ const queryClient = useQueryClient()
 const text = useText()
 const EnsoDevtools = reactComponent(EnsoDevToolsReact)
 const ReactQueryDevtools = reactComponent(ReactQueryDevtoolsReact)
+const debugHoverAreas = useFeatureFlag('debugHoverAreas')
+useAppClass(() => ({ debugHoverAreas: debugHoverAreas.value }))
```

Comment:
`ProtectedLayout` is a strange place for this one. I would either put it in `App` or in `AppContainer`

Reply:
This is because `ProtectedLayout` renders `EnsoDevtools`, and is effectively the closest Vue parent. This functionality is related to devtools (react), but needs to run in vue context and needs to run even when devtools are not currently displayed.